assets/component-mega-menu.css

```diff
@@ -62,10 +62,6 @@
   flex-wrap: wrap;
 }
 
-.header--top-center .mega-menu__list > li {
-  width: 16%;
-}
-
```

Comment:
👀 Reverted it back to 16% and removed the `gap` since we are using flex now. Added margin instead. 16% makes sense since 100% / 6 columns. 

https://github.com/Shopify/dawn/pull/1482/commits/2184539ce75b0ef67ee81683cce0bb9d01b00774

Reply:
Note: the padding is for longer text, but can definitely be removed 🤔 thoughts?

Reply:
.... I was testing with `130%` body font size scale.

Reply:
Good point about the larger body font size 🤔 keeping the padding